src/_actions/AccountActions.js

```diff
@@ -14,3 +14,17 @@ export const serverDataPayoutCurrencies = serverResponse => ({
     type: types.SERVER_DATA_PAYOUT_CURRENCIES,
     serverResponse,
 });
+
+export const serverAuthFailed = serverResponse => ({
```

Comment:
It's my personal preference, I prefer actions to be more specific, instead of having most actions looks like generic setters, so when looking at caller I'll know what's going on directly.

anyway, I do not have strong opinion on it, so I'll change it?